src/overview/search-bar/actions.ts

```diff
@@ -101,7 +103,12 @@ export const search: (args?: any) => Thunk = (
         return
     }
 
-    dispatch(sidebarActs.closeSidebar())
+    const isLoggableURL = isLoggable({ url: window.location.href })
+
+    if (!isLoggableURL) {
```

Comment:
I didn't realize I thought ribbon is available only on loggable pages

Reply:
Yeah, I think there is some check like that somewhere. So basically these lines just differentiate if search is being done on the overview, and if so it closes the sidebar, right? If so, I feel like there could be a simpler way rather than checking the URL (i.e., add an opt flag argument to the thunk), but if not let's put a comment here explaining what's going on.